Fix grammar (#2078)
flyteorg/flyte: rsts/concepts/nodes.rst

Nodes
=====

> A node represents a unit of execution or work within a workflow. Ordinarily, a node encapsulate an instance of 

Reviewer: ```suggestion
A node represents a unit of execution or work within a workflow. Ordinarily, a node encapsulates an instance of 
```

Workflow Nodes
--------------

> A node can contain an entire sub-workflow. Since workflow executions always require a launch plan, workflow nodes can reference a launch plan that would trigger their enclosed workflows.

Reviewer: ```suggestion
A node can contain an entire sub-workflow. Since workflow executions always require a launch plan, workflow nodes have a reference to a launch plan to trigger their enclosed workflows.
```
